wg-qa/README.md

```diff
@@ -0,0 +1,28 @@
+# QA Working Group
+
+Oversees stability, quality, and testing criteria within Electron.
+
+## Membership
+
+| Avatar| Name | Role | Time Zone |
+| ----------------- | ----------------- | ----------------- | ----------------- |
+| <img src="https://github.com/MariaBeliaeva.png" width=100 alt="@MariaBeliaeva">  | Maria Beliaeva [@MariaBeliaeva](https://github.com/MariaBeliaeva)| **Chair** | PT (San Francisco) |
+| <img src="https://github.com/VerteDinde.png" width=100 alt="@VerteDinde">  | Keeley Hammond [@VerteDinde](https://github.com/VerteDinde) | Member | PT (Portland) |
+| <img src="https://github.com/cruzerld.png" width=100 alt="@cruzerld">  | David Cruz [@cruzerld](https://github.com/cruzerld) | Member | ET (New York City) |
+| <img src="https://github.com/alexeykuzmin.png" width=100 alt="@alexeykuzmin">  | Alexey Kuzmin [@alexeykuzmin](https://github.com/alexeykuzmin) | Member | CET (Prague) |
+| <img src="https://github.com/sofianguy.png" width=100 alt="@sofianguy"> | Sofia Nguy [@sofianguy](https://github.com/sofianguy) | Member | PT (San Francisco) |
+
+
+
+## Parent Group
+
+* Releases WG
+
```

Comment:
Above Areas of Responsibility, we need to add our OKRs from the post-summit meeting. Like so:

```
## Current Objective and Key Results
**Objective:**

Electron stable releases should be of high quality and easy for apps to upgrade.

**Key Results:**
* Measure the average or cumulative delta between an electron stable release, and the AFP participants shipping that same stable release in their GA product.
* Measure the average or cumulative delta between an electron stable release, and the AFP participants shipping that same stable release in their canary product.
* Reduce the size of outstanding AFP-raised issues at the time of stable.
* Reduce the number of _would-be blockers_ discovered _after_ a stable ship.

```

Reply:
I would like to see these integrated into the group. Remember, the KRs are not _must haves_ they are just guides to help tell if you actually hit your objective.

May also be worth including a KR about growing out the working group.

Reply:
Added KRs, and with Maria added a growth KR